### DISCUSSION TITLE
Operating systems

### DISCUSSION THREAD

**Discussion by Sure10times** [parallaxsw/OpenSTA#2] [Q&A]
Hello developers of OpenSTA!
Thank you for such a powerfull tool!

Could you tell me please, what Operating systems does "OpenSTA" can be used ? (Appart from Linux(Ubuntu))

Thank you!

**Comment by akashlevy** [comment]
Officially supported: Ubuntu 22.04+ (Docker), CentOS 7+ (Docker), macOS (Homebrew). See the README. You can probably get it to build on most platforms though.
